config/mailinglist/rules/jdk.json

```diff
@@ -222,7 +222,7 @@
             "src/java.xml/",
             "src/jdk.accessibility/",
             "src/jdk.incubator.foreign/",
-            "src/jdk.incubator.jpackage/",
+            "src/jdk.jpackage/",
             "src/jdk.internal.jvmstat/",
```

Comment:
core-libs-dev has been used to discuss the jpackage tool although it's not really core libraries. I've often thought it should have had its own list.

Several other values in core-libs could be pruned:

make/jdk/src/classes/build/tools/jdwpgen is serviceability-dev
src/jdk.accessibility is client libs, swing-dev?
test/jdk/javax/accessibility is client libs, swing-dev?
src/jdk.internal.jvmstat is serviceability-dev
src/jdk.jconsole is serviceability-dev
src/java.smartcardio is security-dev

Reply:
@AlanBateman please file an issue on Skara in JBS and we look into it :+1: